packages/react-core/src/components/Pagination/Pagination.tsx

```diff
@@ -109,6 +109,15 @@ export interface PaginationProps extends React.HTMLProps<HTMLDivElement>, OUIAPr
   dropDirection?: 'up' | 'down';
   /** Page to start at. */
   firstPage?: number;
+  /** @beta Insets at various breakpoints. */
+  inset?: {
```

Comment:
Should a `usePageInsets` prop similar to what Toolbar has also be added? @mcoker

Reply:
Good question @thatblindgeye. I do not see it in the core example.